tools/windows/install-help/cal/customactiondata.cpp

```diff
@@ -252,9 +252,22 @@ void CustomActionData::findSuppliedUserInfo(std::wstring &input, std::wstring &c
 
     if (computed_domain == L".")
     {
-        WcaLog(LOGMSG_STANDARD, "Supplied qualified domain '.', using hostname");
-        computed_domain = machine.GetMachineName();
-        domainUser = false;
+        if (GetTargetMachine().IsDomainController())
+        {
+            // User didn't specify a domain OR didn't specify a user, but we're on a domain controller
```

Comment:
```suggestion
            // User didn't specify a domain OR didn't specify neither a domain nor user, but we're on a domain controller
```
Is this more accurate, or I misunderstood what this code is doing?

Reply:
> I think we can simplify it to say "when the user didn't specify a username on a DC".

What confused me is that here we don't check there's no specified user, only that there's no specified domain (`if (computed_domain == L".")`). Is that done outside this function maybe?